Makefile

```diff
@@ -123,9 +123,10 @@ build: clean tox
 	tar -uf dist/python-kiwi.tar kiwi-${version}/doc/build/latex/kiwi.pdf
 	gzip dist/python-kiwi.tar
 	rm -rf kiwi-${version}
-	# provide rpm changelog from git changelog
-	git log | helper/changelog_generator |\
-		helper/changelog_descending > dist/python-kiwi.changes
+	# update rpm changelog using reference file
+	helper/update_changelog.py package/python-kiwi.changes > \
```

Comment:
Prefix this with `python3` so that it runs with Python 3 instead of requesting Python 2?

Reply:
I've updated the install list to request only py3, doesn't make sense to add more, thanks

Reply:
Not prefixing it with python3 makes the rpm stage fail on Fedora, as there is no `python` in env, there is only `python3`.